gemrb/GUIScripts/iwd2/LoadScreen.py

```diff
@@ -61,17 +58,19 @@ def StartLoadScreen ():
 	
 	def EndLoadScreen ():
 		GemRB.SetVar ("Progress", 0)
-		MessageWindow.UpdateControlStatus()
-		MessageWindow.TMessageTA.Append("[p][color=f1f28d]" + GemRB.GetString (HintStr) + "[/color][/p]\n")
+		TMessageTA = GemRB.GetView("MsgSys", 0)
+
+		TMessageTA.Append("[p][color=f1f28d]" + GemRB.GetString (HintStr) + "[/color][/p]\n")
 
 		Skull = LoadScreen.GetControl (3)
 		Skull.SetMOS ("GTRBPSK2")
-		LoadScreen.SetVisible (WINDOW_VISIBLE)
-		LoadScreen.Unload()
+		
+		LoadScreen.SetAction(lambda win: GemRB.GamePause(0, 0), ACTION_WINDOW_CLOSED)
+		GemRB.SetTimer(lambda: LoadScreen.Close(), 500, 0)
```

Comment:
Codacy found an issue: [Lambda may not be necessary](https://app.codacy.com/manual/gemrb/gemrb/pullRequest?prid=6310305)